gpytorch/lazy/lazy_tensor.py

```diff
@@ -731,6 +731,106 @@ def add_jitter(self, jitter_val=1e-3):
         diag = torch.tensor(jitter_val, dtype=self.dtype, device=self.device)
         return self.add_diag(diag)
 
+    def cat_rows(self, fant_train, fant_fant):
```

Comment:
probably want to use different variables here to be more generic than fantasizing